docs/UserGuide.md

```diff
@@ -424,8 +430,8 @@ Format: `deletenote ni/NOTE_INDEX li/LESSON_INDEX [si/STUDENT_INDEX]`
 * `NOTE_INDEX`, `LESSON_INDEX` and `STUDENT_INDEX` **must be a positive integer** 1, 2, 3, …​
 
 Example:
-* `deletenote ni/1 li/1 si/2` deletes a note for the 2nd student from the 1st lesson.
-* `deletenote ni/1 li/1` deletes a note from the 1st lesson.
+* `deletenote ni/1 li/1 si/2` deletes the 1st note for the 2nd student within the 1st lesson in your lesson list.
```

Comment:
Same as suggestion for `addnote`, I think the phrasing might be more consistent with the explanations for the other commands if you mention the student list.